canva.com.website.json

```diff
@@ -4,26 +4,29 @@
    "serviceId":"website",
    "serviceName":"Canva Websites",
    "syncPubKeyDomain":"domainconnect.canva.com",
-   "version": 2,
-   "logoUrl": "https://static.canva.com/static/images/canva_logo_100x100@2x.png",
+   "version":3,
+   "logoUrl":"https://static.canva.com/static/images/canva_logo_100x100@2x.png",
    "description":"Enables a domain to work with Canva",
    "variableDescription":"verify is the verification value for the site",
    "records":[  
       {  
+         "groupId":"hosting",
```

Comment:
This allows our FE to request only for hosting or verification records to be set. I don't think we'll use it yet, but we may as well add it here while we update the template.

See [here](https://github.com/Domain-Connect/spec/blob/master/Domain%20Connect%20Spec%20Draft.adoc#template-record:~:text=in%20the%20template.-,Group,-Id)
> (optional) This parameter specifies the groups from the template to apply. If no group is specified, all groups are applied. Multiple groups may be specified in a comma delimited format.